src/components/sbb-datepicker-next-day/sbb-datepicker-next-day.tsx

```diff
@@ -96,8 +90,11 @@ export class SbbDatepickerNextDay extends LitElement {
   }
 
   public override willUpdate(changedProperties: PropertyValues<this>): void {
-    if (changedProperties.has('datePicker')) {
-      this._findDatePicker(this.datePicker, changedProperties.get('datePicker'));
+    if (
+      changedProperties.has('datePicker') &&
+      this.datePicker !== changedProperties.get('datePicker')
+    ) {
+      this._init(this.datePicker);
     }
   }
```

Comment:
```suggestion
    if (changedProperties.has('datePicker')) {
      this._init(this.datePicker);
    }
  }
```
Did you try it without the !== check?
I think you don't need it because Lit already does that before calling the willUpdate.
I might be wrong though, so be sure to confirm it